src/Microsoft.AspNet.OData.Shared/DefaultEdmPatchMethodHandler.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.OData.Edm;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// This is the default Patch Handler for non CLR type. This calss has default Get, Create and Update
+    /// and will do these actions. This will be used when the original collection to be Patched is provided.
+    /// </summary>
+    internal class DefaultEdmODataAPIHandler : EdmODataAPIHandler
```

Comment:
>DefaultEdmODataAPIHandler [](http://example.com/codeflow?start=19&length=25)

Do we really have both a DefaultEdmPatchMethodHandler and a DefaultODataAPIHandler?

Reply:
No we dont have any DefaultEdmPatchMethodHandler 
 we have 2 defaults, 1 for typed another for typeles

Reply:
I think I was confused because the name of the file was still "DefaultEdmPatchMethodHandler"